eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -63,7 +63,7 @@ function Login([string]$subscription, [string]$clusterGroup, [switch]$pushImages
 function DeployStressTests(
     [string]$searchDirectory = '.',
     [hashtable]$filters = @{},
-    [string]$environment = 'test',
+    [string]$environment = 'pg',
```

Comment:
Might be helpful to put a quick comment above this parameter explaining what `pg` is short for.